test/js/specs/happy.js

```diff
@@ -1,16 +1,52 @@
 const path = require('path')
+const expect = require('chai').expect;
 import {describe, it} from '../utils/mochaw'
 
 const options = {
   pageObjects: ['DocumentSelection', 'Welcome', 'DocumentUpload']
 }
 
-describe('Happy Paths',options, ({driver,pageObjects}) => {
+const localhostUrl = 'https://localhost:8080/'
+
+describe('Happy Paths',options, ({driver,$,pageObjects}) => {
   const {documentSelection, welcome, documentUpload} = pageObjects
 
+  //Welcome Screen Tests
+  it('test website title', async () => {
+    await driver.get(localhostUrl)
```

Comment:
Do we need to call await driver.get(localhostUrl). Can we just do it once and test multiple elements in the same screen?

Reply:
I just checked that and we can actually do that :)
I think it will be possible only with these tests as if we will need to go through the different screens in the other tests, we will need to start the session from scratch.